src/components/Information.js

```diff
@@ -0,0 +1,246 @@
+              aria-current="step"
+            >
+              <span
+                className="h-2.5 w-2.5 rounded-full bg-cerulean"
+                aria-hidden="true"
+              ></span>
+              <span className="absolute pt-14 mr-3 text-light-gray w-full h-full text-center font-normal text-xs">
+                Personal Info
+              </span>
+            </div>
+          </li>
+          <li className="relative pr-8 sm:pr-20">
+            <div
+              className="absolute inset-0 flex items-center"
+              aria-hidden="true"
+            >
+              <div className="h-0.5 w-full bg-white"></div>
+            </div>
+            <button
+              onClick={Continue}
+              type="button"
+              className="group relative flex h-8 w-8 items-center justify-center rounded-full border-2 border-gray-300 bg-white hover:border-gray-400"
+            >
+              <span
+                className="h-2.5 w-2.5 rounded-full bg-transparent group-hover:bg-gray-300"
+                aria-hidden="true"
+              ></span>
+              <span className="absolute pt-14 mr-3 text-light-gray w-full h-full text-center font-normal text-xs">
+                Hobbies
+              </span>
+            </button>
+          </li>
+          <li className="relative pr-8 sm:pr-20">
+            <div
+              className="absolute inset-0 flex items-center"
+              aria-hidden="true"
+            >
+              <div className="h-0.5 w-full bg-white"></div>
+            </div>
+            <div className="group relative flex h-8 w-8 items-center justify-center rounded-full border-2 border-gray-300 bg-white hover:border-gray-400">
+              <span
+                className="h-2.5 w-2.5 rounded-full bg-transparent group-hover:bg-gray-300"
+                aria-hidden="true"
+              ></span>
+              <span className="absolute pt-14 text-light-gray w-full h-full text-center font-normal text-xs">
+                Skills
+              </span>
+            </div>
+          </li>
+          <li className="relative pr-8 sm:pr-20">
+            <div className="group relative flex h-8 w-8 items-center justify-center rounded-full border-2 border-gray-300 bg-white hover:border-gray-400">
+              <span
+                className="h-2.5 w-2.5 rounded-full bg-transparent group-hover:bg-gray-300"
+                aria-hidden="true"
+              ></span>
+              <span className="absolute pt-14 text-light-gray w-full h-full text-center font-normal text-xs">
+                Profile Picture
+              </span>
+            </div>
+          </li>
+          <li className="relative">
+            <div
+              className="absolute inset-0 flex items-center"
+              aria-hidden="true"
+            >
+              <div className="h-0.5 w-full bg-gray-200"></div>
+            </div>
+          </li>
+        </ol>
+      </nav>
```

Comment:
It's fine to leave it like this but instead of rendering this nav with the stepper in every step component, you could render it on the parent `SignUp`